triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -535,7 +535,9 @@ class Runner(
       val f: Future[Empty] = client.commandClient
         .submitSingleCommand(req)
       f.map(_ => None).recover {
-        case s: StatusRuntimeException =>
+        case s: StatusRuntimeException if s.getStatus != io.grpc.Status.UNAUTHENTICATED =>
+          // Do not capture UNAUTHENTICATED errors.
```

Comment:
How does this look in the standalone trigger runner?

Reply:
IIUC then we don't hit this in the standalone trigger runner. To test this I've set up the copy trigger example using `daml-sdk-head` from this branch and configered HMAC256 validation. I've then created a short lived token for the trigger runner, started the runner, waited for the token to expire, and then submitted a create that should activate the trigger. The trigger runner then failed as shown below:
```
$ daml trigger --dar .daml/dist/copy-trigger-0.0.1.dar --trigger-name CopyTrigger:copyTrigger --ledger-host localhost --ledger-port 6865 --ledger-party Alice --access-token-file trigger-token.jwt
Reading token from trigger-token.jwt...
Trigger  is running as Alice (context: {triggerDefinition=f2c380ed08917de575b4a7f8bd05128cfb732bc334ecc17524c9b722210117fe:CopyTrigger:copyTrigger})
Exception in thread "main" io.grpc.StatusRuntimeException: PERMISSION_DENIED
        at io.grpc.Status.asRuntimeException(Status.java:533)
        at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:453)
        at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:426)
        at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:689)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$900(ClientCallImpl.java:577)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:751)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:740)
        at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
        at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
daml-helper: Received ExitFailure 1 when running
Raw command: java -Dlogback.configurationFile=/home/aj/.daml/sdk/0.0.0/daml-sdk/trigger-logback.xml -jar /home/aj/.daml/sdk/0.0.0/daml-sdk/daml-sdk.jar trigger --dar .daml/dist/copy-trigger-0.0.1.dar --trigger-name CopyTrigger:copyTrigger --ledger-host localhost --ledger-port 6865 --ledger-party Alice --access-token-file trigger-token.jwt
```
Running the same test on master produces the same error.